packages/nouns-contracts/contracts/client-incentives/Rewards.sol

```diff
@@ -320,11 +320,16 @@ contract Rewards is
         require(lastProposalId >= t.nextProposalIdToReward, 'bad lastProposalId');
         require(isSortedAndNoDuplicates(votingClientIds), 'must be sorted & unique');
 
+        uint16 proposalEligibilityQuorumBps_ = $.params.proposalEligibilityQuorumBps;
+
         NounsDAOTypes.ProposalForRewards[] memory proposals = nounsDAO.proposalDataForRewards(
             t.nextProposalIdToReward,
             lastProposalId,
+            proposalEligibilityQuorumBps_,
```

Comment:
I think the natspec of this function can add more clarity on the constraints we enforce, specific things on my mind:

* what makes a prop eligible: we are now filtering out canceled props
* define the minimum reward period and how it works, highlighting that we do not use block.timestamp
* votingClientIds: clarify that it reverts if they have zero votes on *eligible* props

Reply:
updated natspec: 157d500e5257bcc22fe914f0ceef7e1fe93ea92f